doc/howto/network_increase_bandwidth.md

```diff
@@ -17,7 +17,30 @@ In general, you should use small `txqueuelen` values with slow devices with a hi
 For the `net.core.netdev_max_backlog` value, a good guideline is to use the minimum value of the `net.ipv4.tcp_mem` configuration.
 ```
 
-## Increase the network bandwidth on the LXD host
+## Ubuntu >= 18.04: Increase the network bandwidth on the LXD host
+
+Complete the following steps to increase the network bandwidth on the LXD host:
+
+1. Increase the transmit queue length (`txqueuelen`) of both the real NIC (for example, `enp5s0f1`) and the LXD NIC (for example, `lxdbr0`).
+   To make the change permanent, create file /etc/udev/rules.d/60-custom-txqueuelen.rules with the following:
+
+       KERNEL=="enp5s0f1", RUN+="/sbin/ip link set %k txqueuelen 10000"
+       KERNEL=="lxdbr0", RUN+="/sbin/ip link set %k txqueuelen 10000"
+      
+   Apply the above udev rules via:
+
+       sudo udevadm trigger
+
+2. Increase the receive queue length (`net.core.netdev_max_backlog`).
```

Comment:
```suggestion
1. Increase the receive queue length (`net.core.netdev_max_backlog`).
```
(It will show up as 2. - but using "1." everywhere is cleaner because it will update if you include a step.)